engine-tests/src/test/java/org/terasology/documentation/ApiScraper.java

```diff
@@ -19,6 +21,8 @@
  * Enumerates all classes and packages that are annotated with {@link API}.
  */
 public final class ApiScraper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ApiScraper.class);
```

Comment:
Have we discussed about the `LOGGER` vs `logger` naming before? I personally prefer `logger`, since all capital naming tends to be reserved for constants/literals. This logger object is strictly-speaking a constant but it is used in a non-constant fashion (it performs writes, whereas normal constants only perform reads). I am aware that checkstyle probably disagrees with me on this.